cmd/minishift/cmd/util/registration.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package cmd
+package util
```

Comment:
This is not a util class...

Reply:
I kind of agree with you. I am fine it with keeping it as cmd pkg.

Reply:
But I do not want to make it a separate pkg in cmd. Because it does not make sense to create cmd/registration when registration is a CLI command in minishift. But I can not import cmd in profile as it will lead to 
```
import cycle not allowed
package github.com/minishift/minishift/cmd/minishift
	imports github.com/minishift/minishift/cmd/minishift/cmd
	imports github.com/minishift/minishift/cmd/minishift/cmd/profile
	imports github.com/minishift/minishift/cmd/minishift/cmd
make: *** [/Users/lmohanty/code/gocode/bin/minishift] Error 1
```